src/python/WMCore/Database/MongoDB.py

```diff
@@ -44,19 +45,40 @@ def __init__(self, database=None, server=None, port=None, replicaset=None,
             msg = "You are trying to mock MongoDB, but you do not have mongomock in the python path."
             self.logger.critical(msg)
             raise ImportError(msg)
+
+        # NOTE: We need to explicitely check for server availiability.
```

Comment:
If you can provide a link to where this is documented, I think it would be better than pasting here the whole comment on it.

Reply:
Thanks @amaltaro. I agree it would be better, but this part  in particular is taken from the help() function to the `mongoClt` object. So this is part of the python docstring of the mongo client itself. I did search to find it somewhere online though, but failed to see anything going to that details.